tests/test_image_verifier.py

```diff
@@ -21,179 +21,317 @@
 #
 # Written by Jan Kaluza <jkaluza@redhat.com>
 
-from unittest.mock import MagicMock, patch
+from unittest.mock import MagicMock
 
 from freshmaker.image_verifier import ImageVerifier
 from freshmaker.lightblue import ContainerRepository, ContainerImage
 from tests import helpers
 
 
 class TestImageVerifier(helpers.FreshmakerTestCase):
-
     def setUp(self):
         super(TestImageVerifier, self).setUp()
-        self.lb = MagicMock()
-        self.verifier = ImageVerifier(self.lb)
+        self.pgql = MagicMock()
+        self.verifier = ImageVerifier(self.pgql)
 
     def test_verify_repository_no_repo(self):
-        self.lb.find_container_repositories.return_value = None
+        self.pgql.find_repositories_by_repository_name.return_value = None
         self.assertRaisesRegex(
-            ValueError, r'Cannot get repository.*',
-            self.verifier.verify_repository, "foo/bar")
+            ValueError,
+            r"Cannot get repository.*",
+            self.verifier.verify_repository,
+            "foo/bar",
+        )
 
     def test_get_verify_repository_multiple_repos(self):
-        self.lb.find_container_repositories.return_value = ["foo", "bar"]
+        self.pgql.find_repositories_by_repository_name.return_value = [
+            ContainerRepository(
```

Comment:
@qixiang Is it neessary for repositories to be mocked also with `content_sets` as here for example https://github.com/redhat-exd-rebuilds/freshmaker/blob/pyxis/tests/test_lightblue.py#L572?

Reply:
Unlike the LightBlue APIs, our find_repositories_* APIs from pyxis_gql.py only returns list of dicts, not list of `ContainerRepository`.

Reply:
I guess I should also only return dicts for images, not ContainerImage? @qixiang

Reply:
right, APIs in pyxis_gql.py don't convert results to `ContainerImage` or `ContainerRepository`.